framework/src/main/java/com/github/dbmdz/flusswerk/framework/config/properties/Routing.java

```diff
@@ -101,19 +102,23 @@ public FailurePolicy getFailurePolicy(Message message) {
     return getFailurePolicy(message.getEnvelope().getSource());
   }
 
+  public static Routing defaults() {
+    return new Routing(null, null, null, null);
+  }
+
   public static class FailurePolicyProperties {
 
     private final Integer retries;
     private final String retryRoutingKey;
     private final String failedRoutingKey;
-    private final Integer deadLetterWait;
+    private final Integer backoffMs;
```

Comment:
Nit: Der Name ist nicht ganz Ideal, um das `Ms` zu kapieren muss man erstmal verstehen dass es sich hier um ne Zeit handelt. Wäre da ne `Duration` als Datentyp nicht sowieso besser? Also `Duration backoff`. Dann können User in die Config Werte wie `100ms`, `10s` oder `1y` schreiben.

Siehe hier z.b. in Spring Boot für Redis: https://github.com/spring-projects/spring-boot/blob/master/spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/data/redis/RedisProperties.java#L69-L72

Reply:
Danke, ist geändert.